annotation-processor/pom.xml

```diff
@@ -142,6 +141,34 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.moditect</groupId>
+                <artifactId>moditect-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>add-module-infos</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>add-module-info</goal>
+                        </goals>
+                        <configuration>
+                            <module>
+                                <moduleInfo>
+                                    <name>${java.module.name}</name>
+                                    <exports>
+                                        !org.hibernate.validator.ap.internal;
+                                        !org.hibernate.validator.ap.internal.*;
+                                        *;
+                                    </exports>
+                                </moduleInfo>
+                            </module>
+                            <jdepsExtraArgs>
+                                --multi-release=${maven.compiler.release}
+                            </jdepsExtraArgs>
```

Comment:
Is this absolutely necessary?

If so, please maybe add a comment explaining why, like in Hibernate Search?

```

                                <!--
                                    Running the jdeps tool will fail without this option, if there are any multi-release jars
                                    on the path. And since some modules might have such jars, e.g. util-common has Jandex,
                                    it's safer to add this additional option:
                                -->
```

Reply:
ok I've double-checked it and this was "forced" by the fact AP being multi-release jar... but we don't need that anymore. I've opened https://github.com/hibernate/hibernate-validator/pull/1380 for that